tests/common/aws_test_runner.c

```diff
@@ -230,6 +230,11 @@ static void RunTests( void )
     #if ( testrunnerFULL_COMMON_IO_ENABLED == 1 )
         RUN_TEST_GROUP( Common_IO );
     #endif
+
+    #if ( testrunnerFULL_CORE_MQTT_ENABLED == 1 )
+        RUN_TEST_GROUP( coreMQTT_Integration );
+        RUN_TEST_GROUP( coreMQTT_Integration_AWS_IoT_Compatible );
```

Comment:
It is better to mention that the coreMQTT_Integration test group will work only with a mosquitto broker.

Reply:
I have added a separate config `testrunnerFULL_CORE_MQTT_ENABLED` to separate these 2 test groups

Reply:
Sounds good. Thanks